packages/docs-worklets/docs/memorySharing/_category_.json

```
@@ -0,0 +1,7 @@
+{
+  "label": "Memory Sharing",
```

How about just 'Memory'?

```suggestion
  "label": "Memory",
```